atmo/manifest.yaml

```diff
@@ -0,0 +1,9 @@
+---
+name: Atmo
```

Comment:
name in lower case 
Title  you can add as Atmo 
example https://github.com/civo/kubernetes-marketplace/blob/master/falco/manifest.yaml